utm.yaml

```diff
@@ -983,9 +987,140 @@ components:
+          - specific
+          - certified
+        uas_class:
+          description: |-
+            UAS aircraft class as defined in COMMISSION DELEGATED REGULATION (EU) 2019/945 (C0 to C4)
+            and COMMISSION DELEGATED REGULATION (EU) 2020/1058 (C5 and C6).
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 4.
+          type: string
+          enum:
+          - C0
+          - C1
+          - C2
+          - C3
+          - C4
+          - C5
+          - C6
+          - other
+        uas_type_certificate:
+          description: |-
+            Provisional field. Not applicable as of September 2021.
+
+            Required only if `uas_class` is set to `other` by ANNEX IV of COMMISSION IMPLEMENTING
+            REGULATION (EU) 2021/664, paragraph 4.
+          type: string
+        # TODO: clarify options
+        identification_technologies:
+          description: |-
+            Technology used to identify the UAS.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 6.
+          type: array
+          items:
+            type: string
+          example: ["netrid", "ads-b"]
+        # TODO: clarify options
+        connectivity_methods:
+          description: |-
+            Connectivity methods.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 7.
+          type: array
+          items:
+            type: string
+        endurance:
+          description: |-
+            Endurance of the UAS. This is expressed in minutes.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 8.
+          type: number
+          format: int32
+        emergency_procedure_url:
+          description: |-
+            The URL at which the applicable emergency procedure in case of a loss of command and
+            control link may be retrieved.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 9.
+          type: string
+        # TODO: add standard reference
+        operator_id:
+          description: |-
+            Registration number of the UAS operator.
+
+            The format is defined in EASA Easy Access Rules for Unmanned Aircraft Systems GM1 to AMC1
+            Article 14(6) Registration of UAS operators and ‘certified’ UAS.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 10.
+          type: string
+          example: FIN87astrdge12k8
+        # TODO: add standard reference
+        uas_id:
+          description: |-
+            When applicable, the registration number of the unmanned aircraft.
+
+            This is expressed using the nationality and registration mark of the unmanned aircraft in
+            line with ICAO Annex 7.
+
+            Required by ANNEX IV of COMMISSION IMPLEMENTING REGULATION (EU) 2021/664, paragraph 10.
```

Comment:
Should this be a required field?

Reply:
This one is optional.

Reply:
It seems like maybe we have the wrong verb here then?  This comment indicates `Required by [...]` -- maybe that should be something like `Recommended by [...]` or `Specified by [...]`?

Reply:
Thank you, change to `Specified by`